src/pymoca/backends/casadi/api.py

```diff
@@ -56,6 +68,18 @@ def dae_residual_function(self):
     def initial_residual_function(self):
         return self._initial_residual_function
 
+    @property
+    def condition_relations(self):
+        raise NotImplementedError
+
+    @property
+    def reinit_equations(self):
+        raise NotImplementedError
+
+    @property
+    def discrete_update(self):
+        raise NotImplementedError
```

Comment:
The above three need to be implemented for caching of hybrid systems models to work.